geojson_pydantic/features.py

```diff
@@ -34,6 +34,24 @@ def set_geometry(cls, v):
             return v.__geo_interface__
         return v
 
+    @property
+    def __geo_interface__(self) -> Dict:
+        """GeoJSON-like protocol for geo-spatial (GIS) vector data.
+
+        ref: https://gist.github.com/sgillies/2217756#__geo_interface
+        """
+        geo: Dict[str, Any] = {"type": self.type, "geometry": self.geometry}
```

Comment:
Doesn't this need to call `.dict()` eventually?  `self.geometry` here is a pydantic geometry model not a dictionary which I think would violate geo interface spec?

Reply:
Maybe should use `geom.__geo_interface__` to get the geometry instead, like how its done in feature collections.

Reply:
oh I think it should call `geometry.__geo_interface__`